### PR TITLE
Styling, optimization changes

### PR DESCRIPTION
**Pull request by TeresaGobble**

### CODE REVIEW

File: client/App.js [hunk 1]
```diff
@@ -313,11 +313,12 @@ const App = () => {
         axios.get(`/products/${currentProductID}/styles/`)
         .then((res) => {
             setCurrentStyle(res.data);
+            console.log("this is the current style", res.data)
```
Comment (clwilshire) [inline client/App.js:316]: 🤔

File: client/components/Related-items-and-comparisons/Card.jsx [hunk 2]
```diff
@@ -10,18 +10,6 @@ const Card = (props) => {
 
   if(props.relatedProductID !== undefined) {
```
Comment (clwilshire) [inline client/components/Related-items-and-comparisons/Card.jsx:11]: is this if statement breaking the app? Looks like the closing curly bracket is now commented out on line 23

**Review by clwilshire — COMMENTED** [review]
Check 2 comments

**Review by yellowstrings — APPROVED** [review]
needs more chi chan :/ 6/10

**Review by clwilshire — COMMENTED** [review]
Looks great! Nice console.log!

**Pull request merged**